scrapy/core/downloader/handlers/ftp.py

```diff
@@ -106,7 +105,7 @@ def _build_response(self, result, request, protocol):
         protocol.close()
         body = protocol.filename or protocol.body.read()
         headers = {"local filename": protocol.filename or '', "size": protocol.size}
-        return respcls(url=request.url, status=200, body=to_bytes(body), headers=headers)
+        return respcls(url=request.url, status=200, body=bytes(body, 'utf-8'), headers=headers)
```

Comment:
why not `body.encode('utf-8')`? Maybe it is just me, but I had to look up what the second argument of bytes does. If was thinking you're doing so to passthrough body if body is already bytes, but this is not what `bytes(body, 'utf-8')` does.

Also, was to_bytes necessary before, i.e. were there cases when body can be bytes, not unicode?

Reply:
> Also, was to_bytes necessary before, i.e. were there cases when body can be bytes, not unicode?

Yes, it looks like protocol.filename is user-defined and likely to be a string, but protocol.body is always bytes, and either can end up as the value for `body`.